rocket_twin/systems/tank/tank_fuel.py

```diff
@@ -0,0 +1,43 @@
+from cosapp.base import System
+
+
+class TankFuel(System):
+    """A simple model of a fuel tank.
+
+    Inputs
+    ------
+    w_in [kg/s]: float,
+        mass flow of fuel entering the tank
+    w_command: float,
+        fuel exit flux control. 0 means tank exit fully closed, 1 means fully open
+
+    Outputs
+    ------
+    w_out [kg/s]: float,
+        mass flow of fuel exiting the tank
+    """
+
+    def setup(self):
+
+        # Fuel
+        self.add_inward("weight_s", 1.0, desc="Structure weight", unit="kg")
```

Comment:
should be removed since you have now a TankGeom that compute the weight ?

Reply:
Check